internal/libpcap/inject.go

```diff
@@ -162,7 +162,7 @@ func InjectFilter(program *ebpf.ProgramSpec, filterExpr string) (err error) {
 		// stack area is safe to use. Here we use stack from -40
 		// because -32, -24, -16 are reserved for pcap-filter ebpf, see
 		// the comments in compile.go
-		StackOffset: 40,
+		StackOffset: 32,
```

Comment:
I think you're right.

`StackOffset: 32` means cloudfare/cbpfc is able to use stack area from r10-32 downwards, in another word, the first available stack pointer points to r10-40.  Your change made it in line with the real situation, thanks.